### PR TITLE
PAMS-142 and PAMS-174 changes

### PR DESCRIPTION
**Pull request by edemiraydin**
Hi Tony,

For PAMS-142, it prints as

Applications:
d2cce
89b2a
ed94a
f2558
Aliases/Collections:
Alias : 89b2a_all Collection(s) : 89b2a_1529460182
Alias : 89b2a_latest Collection(s) : 89b2a_1529460182
Alias : d2cce_all Collection(s) : d2cce_1529460182
Alias : d2cce_latest Collection(s) : d2cce_1529460182
Alias : ed94a_all Collection(s) : ed94a_1529460182
Alias : ed94a_latest Collection(s) : ed94a_1529460182
Alias : f2558_all Collection(s) : f2558_1529460182
Alias : f2558_latest Collection(s) : f2558_1529460182

when --verbose is specified. Please let me know if this is not what you had in mind. Thanks.

### CODE REVIEW

File: collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerCliParser.scala [hunk 1]
```diff
@@ -33,5 +33,10 @@ class CollectionRollerCliArgsParser(args: Seq[String]) extends ScallopConf(args)
     opt[String]("delete-applications", required = false, descr = "Delete applications (operation)")
   lazy val listApplications =
     opt[Boolean]("list-applications", required = false, descr = "List all applications (operation)")
+  lazy val verbose =
+    opt[Boolean]("verbose",
+                 required = false,
+                 descr =
+                   "List addtnl info (aliases and collections) for all applications (operation)")
```
Comment (afoerster) [inline collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerCliParser.scala]: nit spell out 'additional'

File: collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerMain.scala [hunk 2]
```diff
@@ -79,6 +81,28 @@ object CollectionRollerMain extends LazyLogging {
...
+    logger.info("Starting Collection Roller List App")
+
+    val (solr, solrService, collectionRoller) = createServices(parsedArgs)
+    try {
+
+      println("Applications:")
+      collectionRoller.collectionList().foreach(println)
+      println("Aliases/Collections:")
+
+      for ((k, v) <- collection.mutable.LinkedHashMap(
```
Comment (afoerster) [inline collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerMain.scala]: I think this output format might be more readable:

```
appname
    alias_latest
        coll7
    alias_all
        coll1
        coll3
        coll6
```

what do you think?

Comment (afoerster) [inline collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerMain.scala]: I don't think the collection needs to be turned back into a map here, could just leave it as a sequence of pairs

Comment (edemiraydin) [reply collection-roller/src/main/scala/io/phdata/pulse/collectionroller/CollectionRollerMain.scala]: This looks good. I will list it that way.

**Comment by edemiraydin** [comment]
Printing as 

7d633
	7d633_latest
		7d633_1529810895
	7d633_all
		7d633_1529810895
18236
	18236_latest
		18236_1529810895
	18236_all
		18236_1529810895
5b91c
	5b91c_latest
		5b91c_1529810895
	5b91c_all
		5b91c_1529810895
aa129
	aa129_latest
		aa129_1529810895
	aa129_all
		aa129_1529810895

now. Please let me know if anything else needs to change. 

Thanks.

**Comment by afoerster** [comment]
Looks good Eda, thanks!

**Comment by afoerster** [comment]
I'll mark 142 closed, I didn't see anything for 174

**Comment by edemiraydin** [comment]
Hmm.. Looks like you already merged my 174 changes into master --> https://github.com/phdata/pulse/blob/master/alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineImplTest.scala   (See lines 209-221)

Not sure if that's what you meant by "The case classes should be built directly". 

Thanks!

**Pull request merged**